metadata.rb

```diff
@@ -4,9 +4,11 @@
 license          "Apache 2.0"
 description      "Installs/Configures ucarp"
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "1.0.2"
+version          "1.0.3"
```

Comment:
I thought this was required to deploy the changes, since we are changing the `metadata.rb` to include `apt`. 

Do we not do that ?